idaes/core/initialization/initializer_base.py

```diff
@@ -110,7 +108,17 @@ def __init__(self, **kwargs):
 
     def __init_subclass__(cls, **kwargs):
         super().__init_subclass__(**kwargs)
-        cls.__doc__ = add_docstring_list(cls.__doc__ if cls.__doc__ else "", cls.CONFIG)
+        cls.__doc__ = cls.__doc__ + cls.CONFIG.generate_documentation(
```

Comment:
This is missing the "Args:" header for google-style documentation.  Alternatively, you could switch to using numpydoc and the built-in `format='numpydoc'` formatter.